profiles/easy

```diff
@@ -184,16 +184,16 @@ org.kde.kded.smart.smartctl                                     auth_admin:auth_
 org.kde.kinfocenter.dmidecode.systeminformation                 no:yes:yes
 
 # systemd (bsc#641924)
-org.freedesktop.hostname1.set-hostname                          auth_admin
-org.freedesktop.hostname1.set-static-hostname                   auth_admin
+org.freedesktop.hostname1.set-hostname                          auth_admin_keep:auth_admin_keep:yes
+org.freedesktop.hostname1.set-static-hostname                   auth_admin_keep:auth_admin_keep:yes
```

Comment:
Please remove those two. For normal users this should be a rare operation for which entering the root password is okay.

Reply:
Done :+1: